build/do-configNalu_release

```diff
@@ -14,6 +15,7 @@ cmake \
   -DYAML_DIR:PATH=$yaml_install_dir \
   -DENABLE_INSTALL:BOOL=OFF \
   -DCMAKE_BUILD_TYPE=RELEASE \
+  -DOpenFAST_DIR:PATH=openfast_install_dir \
```

Comment:
Can we also explicitly add `-DNALU_USES_OPENFAST:BOOL=ON` here, so that users can disable that if they are not prepared to install OpenFAST as yet.